.github/pull_request_template.md

```diff
@@ -1,17 +1,24 @@
 ## What does this change?
 
-<!--delete and update as appropriate-->
+<!-- Add a description of what your PR changes here -->
 
 ### Common Content?
 
+<!-- This PR adds content to the common-content module -->
+
 Block/s
 
 ### Common Theme?
 
+<!-- This PR adds a feature or bugfix to the common-theme module.
+Please reference the ticket you are addressing .-->
+
 Issue number: #issue-number
 
 ### Org Content?
 
+<!--This PR changes a whole module, a sprint, a page, or a block on a single organisation's module.  -->
+
 Module | Sprint | Page Type | Block Type
```

Comment:
```suggestion
- [ ] Module
- [ ] Sprint
- [ ] Page Type
- [ ] Block Type
```

Reply:
No tbh I want them to write what it's changing, not checklist it, just so I know where to look

Reply:
Fair 👍🏻